server/lib/errors/Unauthorized.js

```diff
@@ -29,7 +29,9 @@ const util = require('util');
  *
  * @requires util
  */
-function Unauthorized(description) {
+
+function Unauthorized(description, key) {
+  'use strict';
```

Comment:
There is not need to add `use strict` anymore.  We are forcing the server to run in strict mode via an npm package.  It also shouldn't be necessary if you are running the latest NodeJS.